modules/hal_nordic/nrf_802154/CMakeLists.txt

```diff
@@ -98,7 +91,6 @@ else()
   target_compile_definitions(zephyr-802154-interface INTERFACE NRF_802154_INTERNAL_RADIO_IRQ_HANDLING=0)
 endif()
 
-set(NRF52_SERIES  ${CONFIG_SOC_SERIES_NRF52X})
```

Comment:
This change does not seem to be related. It would be better to move it to a separate commit.

Reply:
Removed the change from PR because it is not related to the PR topic.